### PR TITLE
fixes #14459 - remove Katello from notification names

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: db/migrate/20160404132250_remove_katello_from_notification_name.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class RemoveKatelloFromNotificationName < ActiveRecord::Migration
+  class FakeMailNotification < ActiveRecord::Base
+    self.table_name = 'mail_notifications'
+  end
+
+  def up
+    FakeMailNotification.all.each do |notification|
+      if notification_names.keys.include?(notification.name)
+        new_name = notification_names[notification.name]
+        FakeMailNotification.where(:name => new_name).destroy_all
```
Comment (ehelms) [inline db/migrate/20160404132250_remove_katello_from_notification_name.rb]: Why destroy all existing ones with the new names?

**Comment by ehelms** [comment]
One comment, and tests failing, otherwise looks good to me

File: db/migrate/20160404132250_remove_katello_from_notification_name.rb (hunk 1 above)
Comment (stbenjam) [inline db/migrate/20160404132250_remove_katello_from_notification_name.rb]: If the user has katello_promote_errata already it'll be assigned to users.  If they run seeds before migration they'll have katello_promote_errata and the new promote_errata, we need to delete that one.

**Comment by stbenjam** [comment]
Tests are green :green_apple:

**Comment by ehelms** [comment]
ACK

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**